log/handler.go

```diff
@@ -170,104 +170,42 @@ func (l *leveler) Level() slog.Level {
 }
 
 func JsonHandler(wr io.Writer) slog.Handler {
-	return &builtinHandler{slog.NewJSONHandler(wr, &slog.HandlerOptions{
+	return slog.NewJSONHandler(wr, &slog.HandlerOptions{
```

Comment:
Hm,this `JsonHandler` method is not used, our code uses
```
		handler = slog.NewJSONHandler(output, nil)
```
Which means that geth doesn't actually use the ReplaceAttr functionality for json at all,

Reply:
oof